src/gui/include/gui/gui.h

```diff
@@ -812,6 +815,8 @@ class Gui
   std::unique_ptr<PlacementDensityDataSource> placement_density_heat_map_;
 
   static Gui* singleton_;
+
+  std::string main_window_title_ = "OpenROAD";
```

Comment:
should belong with the `Gui` class to be consistent?

Reply:
This is the `Gui` class. It can't be on the MainWindow class (if that's what you meant to suggest) because we want to be able to set it before the main window is created

Reply:
You're right, I got distracted by the singleton above it.